examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    cfg.sim.start_time=today.mjd2000 * pk.DAY2SEC
</pre><p>-&gt; </p><pre class="ql-syntax" spellcheck="false">cfg.sim.start_time=today.mjd2000 * pk.DAY2SEC # convert epoch to seconds
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>